libs/features/pages/home/src/lib/components/home-hero-section/home-hero-section.component.scss

```diff
@@ -0,0 +1,63 @@
+
+.container {
+  background: #F9F8F4 0% 0% no-repeat padding-box;
```

Comment:
For reuse of colors, use CSS variables specified in elements/theming/_variables.scss for specifying colors/background colors